astroplan/plots/sky.py

```diff
@@ -0,0 +1,195 @@
+
+    Returns
+    -------
+    An `Axes` object (ax) with a map of the sky.
+
+    Notes
+    -----
+    Coordinate defaults:
+
+        Altazimuth (local horizon) coordinate system.  North is always at top
+        of plot, South is always at the bottom, E/W can be right or left
+        depending on the `north_to_east_cw` option.
+
+        Altitude: 90 degrees (zenith) is at plot origin (center) and 0 degrees
+        (horizon) is at plot edge.  This cannot be changed by user.
+
+        Azimuth: 0 degrees is at North (top of plot), 90 degrees at East, etc.
+        DANGER: Azimuth labels can be changed by user via the `az_label_offset`
+        option, but it is not recommended, as to do so makes it seem as if
+        N/E/S/W are being decoupled from the definition of azimuth
+        (North from az = 0 deg., East from az = 90 deg., etc.).
+
+
+    TODO:
+        1) Add time/date and target name labels?
+    """
+
+    # Set up axes & plot styles if needed.
+    if ax is None:
+        # axisbg option sets background color within plot bounds (circle).
+        # May need to set a color outside of plot bounds.
+        ax = plt.gca(projection='polar')
+        #ax = plt.gca(polar=True)
+    if style_kwargs is None:
+        style_kwargs = {}
+    style_kwargs = dict(style_kwargs)
+    style_kwargs.setdefault('marker', 'o')
+
+    # Grab altitude and azimuth from Astroplan objects.
+    # Note that values must be made dimensionless before plotting.
+    # Modifying altitude is easier than inverting r-axis.
+    altitude = (91 * u.deg - observer.altaz(time, target).alt) *(1/u.deg)
+    # Azimuth MUST be given to plot() in radians.
+    azimuth = observer.altaz(time, target).az * (1/u.deg) * (np.pi/180.0)
+
+    # Some checks & info for labels.
+    if not hasattr(target, 'name'):
+        target_name = '?'
+    else:
+        target_name = target.name
+    style_kwargs.setdefault('label', target_name)
+
+    # We only want to plot positions above the horizon.
+    az_plot = None
+    for alt in range(0, len(altitude)):
+        if altitude[alt] > 91.0:
+            print("Warning: Target " + str(target_name) +
+                  " is below the horizon at time: " + str(time[alt]))
+        else:
+            if az_plot is None:
+                az_plot = np.array([azimuth[alt]])
+            else:
+                az_plot = np.append(az_plot, azimuth[alt])
+    alt_plot = altitude[altitude <= 91.0]
+    if az_plot is None:
+        az_plot = []
+
+    # More axes set-up.
+    # Position of azimuth = 0 (data, not label).
+    ax.set_theta_zero_location('N')
+
+    # Direction of azimuth increase. Clockwise is -1
+    if north_to_east_ccw is False:
+        ax.set_theta_direction(-1)
+
+    # Plot target coordinates.
+    # ax.plot connects dots.
+    #ax.plot(az_plot, alt_plot, **style_kwargs)
+    # ax.scatter does not connect dots.
+    ax.scatter(az_plot, alt_plot, **style_kwargs)
+    #ax.polar(az_plot, alt_plot, **style_kwargs)
```

Comment:
This comment looks like leftover testing code (and also the ax.plot call above), right?  May as well remove it.